### DISCUSSION TITLE
Increase token supply - AEIP18

### DISCUSSION THREAD

**Discussion by samuelmanzanera** [archethic-foundation/aeip#50] [AEIP]
This is a discussion for the AEIP18 located at: https://github.com/archethic-foundation/aeip/blob/main/AEIP-18.md

**Comment by samuelmanzanera** [comment]
I have few remarks

> Currently this is not possible to have this kind of token on Archethic since they will be minted in differents transactions, and so the UTXO will have a different token_address

We should explain a bit more here, because we can actually increase the supply of a token but it will result in multiple UTXO from different reference addresses, like if it was a different token.

> allow_mint is a boolean (true / false), default value is false if property is not set

We should probably indicate if this property is not defined, we can still increase the token supply but creating multiple UTXOs

**Reply by samuelmanzanera** [reply]
So to resolve this confusion, we shouldnt allow default value then for the `allow_mint`. 
It should be required and mandatory to be AEIP18 compliant. 
The default mention should be omitted then

**Reply by Neylix** [reply]
Yep, commited update here bcfeaf165d453123eed8b7662eb6dbd45ef12978
